packages/renderer/src/lib/kube/details/KubePort.svelte

```diff
@@ -59,11 +59,13 @@ async function removePortForward(): Promise<void> {
   if (!mapping) return;
   if (!forwardConfig) return;
   loading = true;
+  error = undefined;
 
   try {
     await window.deleteKubernetesPortForward(forwardConfig);
   } catch (err: unknown) {
     console.error(err);
```

Comment:
Do we want to keep to error on the console, as it is now displayed in the UI?

Reply:
I would argue yes, because traces, when we display to the UI, we might lose information because of stringify, I am not sure what to follow here, we don't have much guidelines on this point